src/autocomplete.js

```diff
@@ -490,6 +487,36 @@ export default class Autocomplete extends Component {
       inputClassList.push(inputClasses)
     }
 
+    const menuClassName = `${cssNamespace}__menu`
+    const menuModifierDisplayMenu = `${menuClassName}--${displayMenu}`
+    const menuIsVisible = menuOpen || showNoOptionsFound
+    const menuModifierVisibility = `${menuClassName}--${(menuIsVisible) ? 'visible' : 'hidden'}`
+
+    const safeMenuAttributes = sanitiseMenuAttributes(menuAttributes || {})
+
+    const menuClassList = [
+      menuClassName,
+      menuModifierDisplayMenu,
+      menuModifierVisibility
+    ]
+
+    // Both React and Preact support the `className` attribute for setting
+    // an element's class in JSX
+    if (safeMenuAttributes.className) {
```

Comment:
Over in https://github.com/alphagov/accessible-autocomplete/pull/649 we talked about empty string `''` values being valid

We should probably allow that here too?

```suggestion
    if ('className' in safeMenuAttributes) {
```

Reply:
Empty strings would add extra spaces to the attribute for nothing, so I think we're better off excluding them. Good point about consistency, though, I'll have a check at how `inputClasses` handles them (`hintClasses` filters them out through the `filter(Boolean)`).

Reply:
>I'm not sure I follow there. Do you have any specifics in mind?

Nothing specific, just the PR description called out consistency as a feature:

>In addition to the update to `menuAttributes`, the PR also adds a `menuClasses`. Even if it doubles with with `menuAttributes.(class|className)`, it provides a consistent API with `inputClasses` and `hintClasses` from #649.

For example, where we destructure `this.props` into `menuAttributes`

Users would consistently think `{ menuAttributes: { className: '' } }` resets the class

Reply:
Oh, I see. I meant consistency in terms of naming, so I'll clarify that in the PR. If `hintClasses` clearing its default on `''` causes trouble with `menuClasses` or `menuAttributes.className` not doing the same, we can update when we get feedback 😊